Articles/Blog/HelloHeatEquation.md

```diff
@@ -1,143 +1,99 @@
+model solar heating of the Earth, even in this relatively simple form,
+interesting and meaningful science questions may be investigated involving
+both transient and steady-state behavior. Examples are determining if water
+pipes embedded in log cabin (solid wood) walls would freeze during a cold
+storm or the required thickness of insulation around fuel pipeline to avoid
+combustion from a fast burning brush fire. These and many other questions
+involving heat flow can be investigated with this relatively simply form of
+the heat equation.
+
+Solutions to the one dimensional heat equation are also very easy for
+beginners to visualize. This can help them understand how different
+approaches work. The simplicity of the physics and mathematics also
+allows beginners to focus more on the implementation details; the
+programming language, parallel execution paradigm, discretization and
+mathematical support libraries.
+
+## A Hello CSE World Repository
+We are announcing the beginnings of an effort to populate and maintain a
+repository of implementations of the heat equation demonstrating various
+aspects of CSE software. The resulting repository will be useful in training
+about fundamental aspects of the design and development of CSE software as
+well as provide examples of continuous integration testing including
+demonstrating the use of ECP-CI resources.
+
+Each implementation we gather into this repository will have to conform
+to a *minimal* set of requirements including providing a fully self-contained
+build, accommodating a common set of command-line arguments, producing ascii
+text files for results output and providing time and space performance data via 
+tools such as [PAPI](https://icl.utk.edu/papi/) or [Caliper](https://software.llnl.gov/Caliper/).
+
+Our goal in collecting such examples isn't so much to demonstrate the use of
+different programming languages as it is to demonstrate different approaches
+to computation and execution paradigms as well as provide instructive insights
+by comparing of time, space and scaling performance of various approaches.
 
 ## Conclusion
-The Heat Equation is well known and has several practical applications in the fields of Mathematics and Physics. Because of this, it clearly has a wide array of implementations, from a sequential for-loop, to a portable, GPU-ready, advanced numerical library example. This exemplar PDE makes it the perfect "Hello World!" for the scientific software community. 
+The Heat Equation is well known and has many practical applications even in
+its one dimensional form. Because of this, a variety of implementations are
+easily available. This exemplar PDE makes it the perfect "Hello World!" for
+the scientific software community. Collecting a number of working examples
+together with unique implementation characteristics will help to create 
+useful introductory CSE training material. We welcome collaboration.
+To get involved, please contact one of the authors.
```

Comment:
Details on getting involved can be found in the repo's contributing guide, please include a link. Remove 'contact one of the authors'.